composer.json

```diff
@@ -104,6 +104,7 @@
     "suggest": {
         "aws/aws-sdk-php": "Required to use the SQS queue driver and SES mail driver (~2.4).",
         "doctrine/dbal": "Required to rename columns and drop SQLite columns (~2.4).",
+        "fanout/fanout": "Required to use the Fanout broadcast driver (~2.0).",
```

Comment:
I put that line in both the composer.json file and src/Illuminate/Broadcasting/composer.json. Is there another file I was supposed to put it into?